packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -4015,7 +4015,7 @@ The default user profile is based on the [System for Cross-Domain Identity Manag
 | countryCode         | country name component of user's address (`country`)                                                                                 | String     | TRUE            | FALSE    | FALSE      |             |             | [ISO 3166-1 alpha 2 "short" code format](https://tools.ietf.org/html/draft-ietf-scim-core-schema-22#ref-ISO3166) |
 | postalAddress       | mailing address component of user's address                                                                                          | String     | TRUE            | FALSE    | FALSE      |             |             |                                                                                                                  |
 | preferredLanguage   | user's preferred written or spoken languages                                                                                         | String     | TRUE            | FALSE    | FALSE      |             |             | [RFC 7231 Section 5.3.5](https://tools.ietf.org/html/rfc7231#section-5.3.5)                                      |
-| locale              | user's default location for purposes of localizing items such as currency, date time format, numerical representations, etc.         | String     | TRUE            | FALSE    | FALSE      |             |             | See Note for more details.                                                                                       |
+| locale              | user's default location for purposes of localizing items such as currency, date time format, numerical representations, and so on.         | String     | FALSE (default)            | FALSE    | FALSE      |             |             | See Note for more details.                                                                                       |
```

Comment:
```suggestion
| locale              | user's default location for purposes of localizing items such as currency, date time format, numerical representations, and so on         | String     | FALSE            | FALSE    | FALSE      |             |             | See [Locale format](#locale-format).                                                                                       |
```
I'm converting the Note to an anchored section for linking. Also, normally we would capitalize the description, even if it's a fragment, but since all the description are lowercase here, I'll leave it. Or we can change it to 
"User's default location for purposes of localizing items such as currency, date time format, numerical representations, and so on."

Reply:
Thanks, made this fix locally.

Reply:
Updated the link and new section. Kept the Nullable = FALSE bit because other parameters were also using this format (should go away when we move to Redocly)